modules/calib3d/src/fundam.cpp

```diff
@@ -1000,7 +1004,7 @@ void cv::convertPointsFromHomogeneous( InputArray _src, OutputArray _dst )
             Point3d* dptr = dst.ptr<Point3d>();
             for( i = 0; i < npoints; i++ )
             {
-                double scale = sptr[i][3] != 0.f ? 1./sptr[i][3] : 1.;
+                double scale = scaleFor(sptr[i][3]);
```

Comment:
build failed due to conversion from `double` (`sptr[i][3]`) to `float` (parameter of `scaleFor()`).

Reply:
added an overloaded version of scaleFor for double